eq/fabric/client.cpp

```diff
@@ -54,31 +54,30 @@ bool Client::connectServer( co::NodePtr server )
     if( server->isConnected( ))
         return true;
 
-    co::ConnectionDescriptionPtr connDesc;
-    if( server->getConnectionDescriptions().empty( ))
-    {
-        connDesc = new co::ConnectionDescription;
-        connDesc->port = EQ_DEFAULT_PORT;
```

Comment:
No, this should stay. This is for the standalone server, which still runs on the hardcoded port.